Makefile

```diff
@@ -26,3 +26,7 @@ openapi:
 openapi-lint:
 	@# Lint each file under openapiv2.
 	find openapiv2 -type f | xargs -I '{}' swagger-cli validate {}
+local-copy-gen:
```

Comment:
I don't think we should put it in the Makefile. Everyone's workspaces are different. The paths you use don't apply to all developers.

Reply:
@donch1989 
> I think everyone can choose their own way. We don't really need this in our contribution guidelines.

I just want to reduce the time for the developers who are not familiar with the codebase.
If you do not think it is a proper way, I have taken it out.
If there are new developers who ask the similar questions, we may think another way to deal with that.

Reply:
The missing part in the guideline is actually something like this, "the GA workflow will generate protogen-go, and the developer should update the protogen-go in the xxx-backend".

About the ways to setup protogen-go locally, for me, this is more like a "how you write the code" problem instead of a "how you contribute this project" problem. This shouldn't block the developer to develop or understand this project. This kind of tutorial should lie in an engineer handbook, not the contribution guideline of this project.

Reply:
> "the GA workflow will generate protogen-go, and the developer should update the protogen-go in the xxx-backend".

I agree on this. It would be better if we have this part.


> About the ways to setup protogen-go locally, for me, this is more like a "how you write the code" problem instead of a "how you contribute this project" problem. 

I understand your point.

Just want to share my journey when I onboarded these repo.
As a new developer who is not familiar with all tech stacks & repo structures here, I was indeed blocked here.

And, I think this part is quite similar with [the contribution part in component](https://github.com/instill-ai/component/blob/main/.github/CONTRIBUTING.md#buidling-vdp).